tests/testrom.c

```diff
@@ -8,6 +8,9 @@
 #define IN_EMULATOR  0
 #endif
 
+#define MEM_START ((void *)0x80000400)
+#define MEM_LENGTH (1024 * 1024 * 4)
```

Comment:
I think entrypoint.S saves the correct size in a global variable you could access, so that this is correct even with the memory pak

Reply:
Actually, it is system.c that does this:
https://github.com/DragonMinded/libdragon/blob/452aa631a5f93a5994faaa6553797d8f99a37bac/src/system.c#L1009-L1013

and uses `get_memory_size` which is here:
https://github.com/DragonMinded/libdragon/blob/1c522f2da5928bb5bb79ac76436c2186091fef75/src/n64sys.c#L172-L175

This takes into account also the CIC being used (while entrypoint.S doesn't... we have an open issue of that)

Reply:
I have made it an absolute linker symbol and also fixed the other bug in the process. Do you have any better ideas for keeping the memory start location?